pkgs/by-name/lu/lutgen/package.nix

```
@@ -8,20 +8,23 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "lutgen";
-  version = "0.12.1";
+  version = "1.0.0";
 
   src = fetchFromGitHub {
     owner = "ozwaldorf";
     repo = "lutgen-rs";
-    rev = "v${version}";
-    hash = "sha256-JFVDiq53AZhiTqEXplCuKoATxeHp/HDJ8p0a6WzVq6I=";
+    rev = "lutgen-v${version}";
+    hash = "sha256-hJ5yD8Yu08kcr2rWY59iVEFJH+chroEWSsP2g5agFuo=";
   };
 
   useFetchCargoVendor = true;
-  cargoHash = "sha256-tOFO2eRWd03zzGX3Ia1IBQ27kmSMX7JgbXmaejRTfp4=";
+  cargoHash = "sha256-VsKRLxh6uRG2A5AvJBMdD+bXg/X9mp5o1iPR9MZhrbQ=";
 
   nativeBuildInputs = [ installShellFiles ];
 
+  cargoBuildFlags = "--bin lutgen";
+  cargoTestFlags = "-p lutgen-cli";
```

nitpick(non-blocking): Strings also seem to work, but most existing code uses lists.

```suggestion
  cargoBuildFlags = [
    "--bin"
    "lutgen"
  ];

  cargoTestFlags = [
    "-p"
    "lutgen-cli"
  ];
```